math/papilo/Portfile

```diff
@@ -6,13 +6,16 @@ PortGroup               cmake 1.1
 PortGroup               github 1.0
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               compilers 1.0
+PortGroup               conflicts_build 1.0
 
 # Boost 1.76 has a bug in multiprecision library,
 # which leads to including x86 header on powerpc.
 boost.version           1.81
+# If this port is active, wrong headers get picked.
+conflicts_build         boost
```

Comment:
this is not great for end-users who might have the "boost" port installed. 

I'm sure there is a way to convince the build systems to pick the correct headers of the requested "boost181" port. Please spend the time needed to accomplish that.